docker/Dockerfile-ubuntu-18.04

```diff
@@ -36,13 +36,13 @@ LABEL maintainer="AzureStack PowerShell Team <azsdevexp@microsoft.com>" \
 
 COPY scripts/* /
 
-RUN pwsh Install-AzureStackPowerShell.ps1 -REPOSITORY ${REPOSITORY} -AZURESTACK_PROFILE ${AZURESTACK_PROFILE} -AZURESTACK_VERSION ${AZURESTACK_VERSION}
+SHELL ["pwsh", "-Command"]
```

Comment:
SHELL ["pwsh", "-Command"]
This switches to powershell instead of cmd so  that you dont need to add pwsh everywhere?

Reply:
Yes